implants/golem/src/main.rs

```diff
@@ -100,17 +100,14 @@ fn main() -> anyhow::Result<()> {
             .build()
             .unwrap();
 
-        let (error_code, result) = match runtime.block_on(run_tomes(parsed_tomes)) {
+        let (error_code, _result) = match runtime.block_on(run_tomes(parsed_tomes)) {
             Ok(response) => (0, response),
             Err(error) => {
                 eprint!("failed to execute tome {:?}", error);
                 (-1, Vec::new())
             }
         };
 
-        if !result.is_empty() {
```

Comment:
Maybe we should just print without the `:?`, that might avoid escaping strings and then we can get rid of the `with_stdout_reporting` API.

Reply:
We need to keep the `with_stdout_reporting` API.
If the tome crashes no output gets printed making it very hard to debug.